examples/example_main.py

```diff
@@ -36,8 +36,8 @@ def __init__(self, wallet_address: int, budget: int = 10_000) -> None:
 
     def action(self, market: Market) -> list[Any]:
         """Implement a custom user strategy"""
-        block_position_list = list(self.wallet.shorts.values())
-        has_opened_short = bool(any((x < -1 for x in block_position_list)))
+        shorts = list(self.wallet.shorts.values())
+        has_opened_short = bool(any((short.balance > 0 for short in shorts)))
```

Comment:
Cool I was just looking at this and I have no idea what that logic was doing before lol. I fixed it differently in my current workflow, but this is good I think.